src/rust/ensogl/example/src/list_view.rs

```diff
@@ -0,0 +1,125 @@
+//! A debug scene which shows the Select Component. The chosen entries are logged in console.
+
+use crate::prelude::*;
+
+use ensogl_core::system::web;
+use ensogl_core::application::Application;
+use ensogl_core::display::object::ObjectOps;
+use ensogl_core::display::shape::*;
+use ensogl_core::display::style::theme;
+use ensogl_core::data::color;
+use ensogl_core::gui;
+use ensogl_text_msdf_sys::run_once_initialized;
+use ensogl_gui_list_view as list_view;
+use logger::enabled::Logger;
+use wasm_bindgen::prelude::*;
+use ensogl_core::display::Scene;
+use ensogl_text::buffer::data::unit::Bytes;
+
+
+
+// ===================
+// === Entry Point ===
+// ===================
+
+/// An entry point.
+#[wasm_bindgen]
+#[allow(dead_code)]
+pub fn entry_point_list_view() {
+    web::forward_panic_hook_to_console();
+    web::set_stdout();
+    web::set_stack_trace_limit();
+    run_once_initialized(|| {
+        let app = Application::new(&web::get_html_element_by_id("root").unwrap());
+        init(&app);
+        mem::forget(app);
+    });
+}
+
+
+
+// ====================
+// === Mock Entries ===
+// ====================
```

Comment:
Not sure how mock are these mock things, but I'd consider placing them in an inline submodule `mock` for separation from non-mock API parts.

Reply:
This is an example, so it is not part of any API

Reply:
ah, ok then